CHANGELOG.md

```diff
@@ -1,21 +1,25 @@
 # Changelog
 
+## [v0.15.0] - 2025-08-28
+
+This release drops support for Go 1.23 and 1.24, and adds support for Go 1.25.
```

Comment:
just revert all changes to this file; we should tackle the release process separately.